### DISCUSSION TITLE
What's the difference between LazyVim/LazyVim and LazyVim/starter?

### DISCUSSION THREAD

**Discussion by mawkler** [LazyVim/LazyVim#166] [Q&A] [answered]
What's the difference between [LazyVim/LazyVim](https://github.com/LazyVim/LazyVim) and [LazyVim/starter](https://github.com/LazyVim/starter/)? Why are there two different configurations and how do I decide which one to use? The [installation instructions](https://www.lazyvim.org/installation) are telling me to clone `LazyVim/starter`, but `LazyVim/LazyVim` seems to have more plugins set up.

**Comment by dpetka2001** [comment, marked as answer]
As far as I can understand `starter` is a template you can use for setting up your configuration. I think `Lazyvim` started off as something that could be used as a configuration, but nowadays is only used as a plugin to provide its functionalities to `starter`. In other words `starter` uses `Lazyvim` in the background to provide all of the functionalities that `Lazyvim` has. Including all of the plugins that are present in `Lazyvim` and some other stuff. You should just clone `starter` and `Lazyvim` will be used as well. Just dive into `Lazyvim` internals if you want to make any customizations. The [site](https://www.lazyvim.org/) provides ample documentation which should get you started.

**Comment by folke** [comment]
@dpetka2001's answer is spot on. To add to this, you don't even need the starter if you don't want to use it. The only thing you really need to use LazyVim, is adding it to your lazy.nvim setup as can be seen in `lua/config/lazy.lua`

**Reply by m1kol** [reply]
So `LazyVim` is just a plugin that `lazy.nvim` package manager downloads and uses?

I was looking into `config` folder and couldn't understand how do all the options and keymaps are set if the corresponding files are empty 🤯 Just couldn't grasp it. There are comments that provided URL to the default options and keymaps are in the corresponding files in this repository... So they are downloaded every time or how does it work, I asked myself?

So in the end they are loaded to the cache as the `LazyVim` plugin is downloaded and then used, right?

**Reply by dpetka2001** [reply]
Lazyvim is installed as a plugin and is stored in the location where `lazy.nvim` stores all the packages it manages. Usually that is `stdpath("data")/lazy/` (in Linux that is usually in `~/.local/share/nvim/lazy/`. Inside there you will see all the downloaded packages and their default configurations. Lazyvim also exists in that directory and you can see the default keymaps/options/autocmds/plugins that Lazyvim comes with.

**Reply by m1kol** [reply]
@dpetka2001 Thanks! Was able to locate it and go through. 👍🏻 

This was the first issue/discussion, where I saw this kind of information, so decided to write right after. After that continued to read and saw similar answers. 😅

**Comment by mawkler** [comment]
@dpetka2001 @folke I see. Thank you!
